atom/browser/autofill/atom_autofill_client.cc

```diff
@@ -121,7 +121,7 @@ IdentityProvider* AtomAutofillClient::GetIdentityProvider() {
 }
 
 ukm::UkmRecorder* AtomAutofillClient::GetUkmRecorder() {
-  return g_browser_process->ukm_recorder();
+  return ukm::UkmRecorder::Get();
```

Comment:
I'm not sure that we want to return a non-null value here. Flagging for privacy review cc @diracdeltas